Model/Ptsv2paymentsProcessingInformationAuthorizationOptionsInitiator.cs

```diff
@@ -60,11 +60,11 @@ public Ptsv2paymentsProcessingInformationAuthorizationOptionsInitiator(string Ty
         public bool? CredentialStoredOnFile { get; set; }
```

Comment:
credentialStoredOnFile is a Y or N field.  This should be change to string in the same way that StoredCredentialUsed has been changed

Reply:
Also see issue reported on 
https://docs.cybersource.com/content/dam/new-documentation/documentation/en/platform-cybs/relnote/all/na/rn-2022-04-15.pdf

Reply:
There is also a similar reported issue here:
https://github.com/CyberSource/cybersource-rest-client-dotnet/issues/129

Reply:
Hi @wrightsonm,

Thank you for bringing this to our attention. I will relay this gap to the relevant team and get it sorted out.

Kindly bear with us for this resolution.

In the meantime, we will release this SDK as provided in this pull request with the promise that we will address your concerns as soon as possible.

-- Gabriel